pages/windows/choco-apikey.md

```diff
@@ -1,20 +1,20 @@
 # choco-apikey
 
 > Manage API keys for Chocolatey sources.
-> More information: <https://chocolatey.org/docs/commands-apikey>.
+> More information: <https://docs.chocolatey.org/en-us/create/commands/api-key>.
 
-- Display a list of sources and their API keys:
+- Print all sources and their API keys:
 
 `choco apikey`
 
-- Display a specific source and its API key:
+- Print the specified source and its API key:
```

Comment:
We try to avoid `its`
```suggestion
- Print a specified source and the associated API key:
```

Reply:
Why?

Reply:
I assume possible issues with your, his, its, etc